UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/MetaTileMap.cs

```diff
@@ -1127,19 +1127,6 @@ public bool IsEmptyAt(Vector3Int position, bool isServer)
 				{
 					return false;
 				}
-
-				if (layer.LayerType == LayerType.Objects)
-				{
-					foreach (RegisterTile o in isServer
-						         ? ((ObjectLayer) LayersValues[index]).ServerObjects.Get(position)
-						         : ((ObjectLayer) LayersValues[index]).ClientObjects.Get(position))
-					{
-						if (o.IsPassable(isServer) == false)
-						{
-							return false;
-						}
-					}
-				}
```

Comment:
Removing this changes the function a lot as objects no longer count?

Reply:
yeah, but the only way to test it thoroughly is to merge it so

Reply:
Im just concerned about how much code relies on the fact that IsEmptyAt assumes there should be no objects not just no tiles.

Reply:
Whats the purpose of removing it?

Reply:
optimisation plus, when the player gets blown up, it checks for matrix and the player finds itself, and therefore thinks it's on the station matrix still, and then when they Fall over that they become IsPassable,  So therefore the at point matrix calculations think it space

Reply:
what's checking for objects?

Reply:
because I think a matrix Shouldn't be determined by objects

Reply:
Was the supermatter and energy ball but looking through what they're doing it seems like it would be fine if objects weren't checked

Could you change the comment above the function then i think it'll be fine to merge